doc/source/examples/integrated_gradients_mnist.ipynb

```diff
@@ -17,6 +17,20 @@
     "A more detailed description of the method can be found [here](https://docs.seldon.io/projects/alibi/en/latest/methods/IntegratedGradients.html). Integrated gradients was originally proposed in Sundararajan et al., [\"Axiomatic Attribution for Deep Networks\"](https://arxiv.org/abs/1703.01365)."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "To enable support for IntegratedGradients, you may need to run\n",
+    "    \n",
+    "```bash\n",
+    "pip install alibi[tensorflow]\n",
+    "```\n",
+    "\n",
+    "</div>"
```

Comment:
Same comment as above, can we make the box title be "Note" and the text part of the body?